src/site/twitch.tv/modules/chat/ChatPubSub.vue

```diff
@@ -103,6 +107,27 @@ async function onModerationAction(msg: PubSubMessageData.ModAction) {
 	}
 }
 
+async function onChatRichEmbed(msg: PubSubMessageData.ChatRichEmbed) {
+	const ctx = msg.twitch_metadata;
+
+	if (!ctx) return;
+
+	// Find the message
+	const message = await messages.awaitMessage(msg.message_id).catch((err) => {
+		log.debug("failed to find new message for chat rich embed", err.message);
+	});
+	if (!message) return;
+
+	const rmsg = ref(message);
```

Comment:
Don't think setting a `ref` here is necessary, this won't really do anything, creating refs like this doesn't introduce reactivity, and the message is already reactive.

Other than that, this PR looks good.